src/index.js

```diff
@@ -403,6 +614,39 @@ export default function ({ types: t }) {
         fixReactNativeViewAttributes(path);
         fixTouchableMixin(t, path);
       },
+      JSXAttribute(path, { opts }) {
+        if (opts.isNewArchitectureEnabled) {
+          extendExistingRef(path);
+        }
+
+        // disable view optimization for only View component
+        if (path.parent.name.name !== 'View') return;
+
+        // must be manually annotated with at least one fs attribute
+        if (
+          path.node.name.name !== 'fsClass' &&
```

Comment:
Should we be defensive here and above with assuming we can always resolve `name.name` without a null check?